posts/inside-rust/2020-05-27-contributor-survey.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title: "2020-05-27 Contributor Survey"
+team: the compiler team <https://www.rust-lang.org/governance/teams/compiler>
+---
+
+You may be aware that [Rust recently turned 5][five]! If you read this blog,
+then you probably already know that the Rust project thrives because of its
+many contributors, and that we're always looking for ways to make it easier for
+people to get involved.
+
+[five]: https://blog.rust-lang.org/2020/05/15/five-years-of-rust.html
+
+Today we are pleased to announce the [Rust contributor survey][survey]. This
+short, anonymous survey should only take a few minutes to fill out, but it will
+really help us in understanding what kinds of problems people experience when
+contributing to Rust so that we can try to address them. We value many
+different kinds of contributions, such as reporting problems, triaging issues,
+organizing meetups, etc, but this survey's purpose is to identify and eliminate
+barriers to entry for code contributions.
+
+The survey is targeting current, past, and future contributors to Rust. We're
+particularly interested in hearing from *new or prospective contributors*.
```

Comment:
Maybe we should include the text that clarifies that -- in this survey -- we're talking about *code contributions*?

Reply:
I updated it to say "hearing from new or prospective code contributors", and we have some discussion in the previous para. Is there some other place I should clarify?